recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "featurewiz" %}
-{% set version = "0.1.91" %}
+{% set version = "0.1.99" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/featurewiz-{{ version }}.tar.gz
```

A detail, but this was a bit inconsistent 
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```